docs/sources/flow/monitoring/agent-resource-usage.md

```diff
@@ -0,0 +1,78 @@
+---
+aliases:
+  - /docs/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/monitor-infrastructure/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/monitor-infrastructure/integrations/agent/flow/monitoring/resource-usage/
+  - /docs/grafana-cloud/send-data/agent/flow/monitoring/resource-usage/
+canonical: https://grafana.com/docs/agent/latest/flow/monitoring/resource-usage/
+description: Guidance for expected Agent resource usage
+headless: true
+title: Resource usage
+---
+
+# {{% param "PRODUCT_NAME" %}} resource usage
+
+This page provides guidance for expected resource usage of 
+{{% param "PRODUCT_NAME" %}} for each telemetry type, based on operational 
+experience of some of the {{% param "PRODUCT_NAME" %}} maintainers.
+
+{{% admonition type="note" %}}
+
+The resource usage depends on the workload, hardware and the configuration used.
+The information on this page is a good starting point for most users, but your
+actual usage may be different.
+
+{{% /admonition %}}
+
+## Prometheus metrics
+
+The Prometheus metrics resource usage depends mainly on the number of active
+series that need to be scraped.
```

Comment:
nit: The Prometheus metrics resource usage depends mainly on the number of active
series that need to be scraped **, and the scrape interval**.